scratch.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
You don't need to check `scratch.py` into git.  Or you can remove it from git with `git rm scratch.py` and the commit it.